crates/bevy_gizmos/src/lib.rs

```diff
@@ -422,14 +422,14 @@ fn prepare_line_gizmo_bind_group(
 ) {
     if let Some(binding) = line_gizmo_uniforms.uniforms().binding() {
         commands.insert_resource(LineGizmoUniformBindgroup {
-            bindgroup: render_device.create_bind_group(&BindGroupDescriptor {
-                entries: &[BindGroupEntry {
+            bindgroup: render_device.create_bind_group(
+                "LineGizmoUniform bindgroup",
+                &line_gizmo_uniform_layout.layout,
+                &[BindGroupEntry {
```

Comment:
With the UniformBuffer/BufferBinding impl you should be able to use it here too now I think?

Reply:
i didn't use it anywhere there was only 1 entry, it doesn't really save any typing. i guess i could for consistency though?

Reply:
Right, makes sense then. I'll personally use it everywhere in my own code but I think it's fine as is.

Reply:
Oof, yeah, nevermind then 😅

Reply:
there's actually quite a few single-entry bindings at zero over the codebase ... i'll try adding a `BindGroupEntries::single` method.

Reply:
added, i think it's nice